### ISSUE TITLE
 object_recognition_node make nothing 

### ISSUE THREAD

**Issue by melikavhf** [tue-robotics/image_recognition#59]
Hi again . 

the .pb and .txt file created successfully by train node https://github.com/tue-robotics/image_recognition/issues/58
![screenshot from 2018-05-28 21-35-07](https://user-images.githubusercontent.com/24391257/40623966-5c5bb764-6299-11e8-952d-3c29adc16e56.png)

now I use  object_recognition_node but nothing created for me 
what should I do ?
![ll](https://user-images.githubusercontent.com/24391257/40624078-fbb01f08-6299-11e8-9958-a22adef51460.png)

and my second question is , it is possible to use this .pb file in object detection package?
https://github.com/cagbal/ros_people_object_detection_tensorflow
for example in this package ?? this package use  Model Zoo of tensorflow object detection and I need to replace my own model . is it possible ? if yes please explain it

**Comment by MatthijsBurgh** [comment]
The object_recogition_node shouldn't be creating anything, it provides ros topics/services. It will probably create some logged files. But that is it.

I don't know if you can use the network in other nodes. I don't think it is possible. But should look that up yourself. The trained network is an inception network.

**Comment by MatthijsBurgh** [comment]
I am not the expert on this, but this is what I know.
There are different networks. Networks which only do recognition. So you provide an image with an object in it. The network will return the type of the object.

The second type of network does detection and recognition. So you provide an image, which can contain nothing or even multiple objects. The network will return which area contains an object and the type of the object.

Of course you can also do these seperate, but I think that all neural networks which do detection, also do recognition. So than you should use an other method to do detection.

I am pretty sure, online you can find a lot of information about both type of networks and also other methods of detection and recognition.

**Comment by reinzor** [comment]
Hi @melikavhf , If you would like to store your estimations / recognitions, use the `save_images` parameter https://github.com/tue-robotics/image_recognition/blob/master/tensorflow_ros/scripts/object_recognition_node#L99

Could you rephrase your question? This package is based on the retrain example as mentioned in the readme. Please use it accordingly, it does not support arbitrary models since they have to match the network architecture

**Comment by reinzor** [comment]
I am going to close this issue since it is not well defined
